cypress/integration/hello-form.spec.js

```diff
@@ -8,10 +8,20 @@ describe('Hello world component', () => {
         cy.visit('/')
 
         // get the input by its test id and type "Hello"
-
+        cy.get('[data-testid="hello-world-input"]')
+            .type('Hello')
         // get the output and insure that it contains "Hello, World!"
-
+        cy.get('[data-testid="hello-world-output"]')
+            .should('contain','Hello, World!')
+        
         // change the text to say "What a wonderful, World!"
+        cy.get('[data-testid="hello-world-input"]')
+            .type('{selectall}{del}What a wonderful')
```

Comment:
Excellent! The code below is in place to catch if you were to fall into the pitfall of typing without first clearing the field. There's another solution that doesn't involve using these built in text manipulation commands as bonus problem if you're interested.